apps/full-site-editing/full-site-editing-plugin/donations/donations.js

```diff
@@ -6,53 +6,104 @@ import classNames from 'classnames';
+	const isTabActive = ( tab ) => activeTab === tab;
+
+	useEffect( () => {
+		if ( ! oneTimePlanId ) {
+			setAttributes( { oneTimePlanId: products[ 'one-time' ] } );
+		}
+	}, [ oneTimePlanId ] );
 
 	return (
-		<div className="donations__container">
+		<Block.div>
 			{ stripeConnectUrl && <StripeNudge stripeConnectUrl={ stripeConnectUrl } /> }
-			<div className="donations__tabs">
-				<Button className={ isActive( 'one-time' ) } onClick={ () => setActiveTab( 'one-time' ) }>
-					{ __( 'One-Time', 'full-site-editing' ) }
-				</Button>
-				<Button className={ isActive( 'monthly' ) } onClick={ () => setActiveTab( 'monthly' ) }>
-					{ __( 'Monthly', 'full-site-editing' ) }
-				</Button>
-				<Button className={ isActive( 'annually' ) } onClick={ () => setActiveTab( 'annually' ) }>
-					{ __( 'Annually', 'full-site-editing' ) }
-				</Button>
-				<div
-					id="donations__tab-one-time"
-					className={ classNames( 'donations__tab', { active: isActive( 'one-time' ) } ) }
-				>
-					{ __( 'One time', 'full-site-editing' ) }
-				</div>
-				<div
-					id="donations__tab-monthly"
-					className={ classNames( 'donations__tab', { active: isActive( 'monthly' ) } ) }
-				>
-					{ __( 'Monthly', 'full-site-editing' ) }
-				</div>
-				<div
-					id="donations__tab-annually"
-					className={ classNames( 'donations__tab', { active: isActive( 'annually' ) } ) }
-				>
-					{ __( 'Annually', 'full-site-editing' ) }
+			<div className="donations__container">
+				{ ( monthlyPlanId || annuallyPlanId ) && (
+					<div className="donations__tabs">
+						<Button
+							className={ classNames( { 'is-active': isTabActive( 'one-time' ) } ) }
+							onClick={ () => setActiveTab( 'one-time' ) }
+						>
+							{ __( 'One-Time', 'full-site-editing' ) }
+						</Button>
+						{ monthlyPlanId && (
+							<Button
+								className={ classNames( { 'is-active': isTabActive( 'monthly' ) } ) }
+								onClick={ () => setActiveTab( 'monthly' ) }
+							>
+								{ __( 'Monthly', 'full-site-editing' ) }
+							</Button>
+						) }
+						{ annuallyPlanId && (
+							<Button
+								className={ classNames( { 'is-active': isTabActive( 'annually' ) } ) }
+								onClick={ () => setActiveTab( 'annually' ) }
+							>
+								{ __( 'Annually', 'full-site-editing' ) }
+							</Button>
+						) }
+					</div>
+				) }
+				<div className="donations__content">
+					<div>
+						{ isTabActive( 'one-time' ) && __( 'Make a one-time donation', 'full-site-editing' ) }
+						{ isTabActive( 'monthly' ) && __( 'Make a monthly donation', 'full-site-editing' ) }
+						{ isTabActive( 'annually' ) && __( 'Make a yearly donation', 'full-site-editing' ) }
+					</div>
+					{ showCustomAmount && (
+						<div>{ __( 'Or enter a custom amount', 'full-site-editing' ) } </div>
+					) }
 				</div>
 			</div>
-		</div>
+			<InspectorControls>
+				<PanelBody title={ __( 'Settings', 'full-site-editing' ) }>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **42** times:
`translate( 'Settings', { context: 'NewDash Page Title'} )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/donation-block-settings)